packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -29,149 +30,48 @@ When using an Okta-hosted flow, you can create a unique sign-in experience by pr
 
-```html
-<img alt="{{orgName}}" src="https://example.com//SomeOtherImage.png" class="org-logo">
-```
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. The console restores the default HTML/CSS and JavaScript code.
 
-### <span v-pre>`{{{errorDescription}}}`</span>
+5. Click **Publish** to commit your changes.
 
-Inserts a detailed description of the error.
+## Use variables
 
-Example:
+The Okta error page template is written using [Mustache](http://mustache.github.io/mustache.5.html) and uses predefined variables to insert relevant values into the error page. To see the variables in a code sample, refer to the error page default code in the code editor. See [Edit the error page](#edit-the-error-page).
 
-```html
-<p class="o-form-explain">What happened? {{{errorDescription}}}</p>
-```
+> **Note:** Variables with double curly braces (`{{`) return escaped HTML by default. Triple curly braces (`{{{`) are used for the `errorDescription` variable to return unescaped HTML. 
 
-> **Note:** Macros with double curly braces return escaped HTML by default. Triple braces `{{{` are used for the `errorDescription` macro to return unescaped HTML. See [Mustache template](http://mustache.github.io/mustache.5.html).
-
-### <span v-pre>`{{back}}`</span>
-
-Inserts the text `Go to Homepage`. When the user selects the button, they are returned to the sign-in page.
-
-Example:
-
-```html
- <a href="/" class="button">{{back}}</a>
-```
-
-### <span v-pre>`{{technicalDetails}}`</span>
-
-Inserts additional error codes, if there are any. See [Okta Error Codes](/docs/reference/error-codes/#okta-error-codes-listed-by-error-code).
-
-Example:
-
-```html
-<p class="technical-details">Additional Error Details: {{technicalDetails}}</p>
-```
-
-> **Note:** The following macros are only available when the Theme Builder feature is enabled in your org.
-
-### <span v-pre>`{{buttonText}}`</span>
-
-Inserts the button text based on the page context. When the user selects the button, they are directed to the `buttonHref` URL. The `{{back}}` macro is also supported for the same purpose.
-
-### <span v-pre>`{{buttonHref}}`</span>
-
-Inserts the hyperlink for the button
-
-Example:
-
-```html
- <a href="{{buttonHref}}" class="button">{{buttonText}}</a>
-```
-
-### <span v-pre>`{{themedStylesUrl}}`</span>
-
-Inserts the URL for the themed style sheet
-
-Example:
-
-```html
- <link href="{{themedStylesUrl}}" rel="stylesheet" type="text/css">
-```
-
-### <span v-pre>`{{faviconUrl}}`</span>
-
-Inserts the URL for the favicon
-
-Example:
-
-```html
- <link rel="shortcut icon" href="{{faviconUrl}}" type="image/x-icon"/>
-```
+| Variable | Description |
+|----------|-------------|
+| <span v-pre>`{{orgName}}`</span> | Inserts the org name title |
```

Comment:
Can we remove "Inserts the" in all the descriptions? It would remove duplication and I think it's implied.

Reply:
@robertnicolo-okta we're talking about predefined variables and what they insert into the error page. How would you communicate that without using the word "inserts"?

Reply:
Putting a pin in this to merge. Will create new Jira for these issues.